hack/kserve_migration/kserve_migration.sh

```
@@ -37,12 +37,11 @@ isControllerRunning() {
         fi
     done
     po_names=$(kubectl get po -n $namespace -o jsonpath='{.items[*].metadata.name}')
-    for po_name in "${prefix}-controller-manager"; do
-        if [ ! -z "${po_names##*$po_name*}" ]; then
-            log ERROR "${prefix} controller services are not installed completely."
-            exit 1;
-        fi
-    done
+    po_name="${prefix}-controller-manager"
+    if [ ! -z "${po_names##*$po_name*}" ]; then
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)